src/main/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSupport.scala

```diff
@@ -37,4 +37,13 @@ trait LibraryServiceSupport {
     (removeOperations ++ updateOperations)
   }
 
+  def updatePublishAttribute(value: Boolean): Seq[AttributeUpdateOperation] = {
+    // TODO: publish attribute can just be a boolean once we support boolean attributes
+    val operations: Seq[AttributeUpdateOperation] =
+    if (value) Seq(AddUpdateAttribute("library:published", AttributeString("true")))
+    else Seq(RemoveAttribute("library:published"))
+
+    operations
```

Comment:
fewer lines of code: this could be just
```
  def updatePublishAttribute(value: Boolean): Seq[AttributeUpdateOperation] = {
    // TODO: publish attribute can just be a boolean once we support boolean attributes
    if (value) Seq(AddUpdateAttribute("library:published", AttributeString("true")))
    else Seq(RemoveAttribute("library:published"))
  }
```